DataRepo/templates/upload.html

```diff
@@ -1,50 +1,109 @@
+                        fields.
+                    </li>
+                    <li>Pay particular attention to the <em>Tracer Compound</em>, <em>Tracer Labeled
+                        Element</em>, <em>Tracer Label Atom Count</em>, <em>Feeding Status</em>, <em>Treatment</em>, and
+                        <em>Study</em> fields.
+                    </li>
+                </ol>
+                </div>
+            </li>
+            <li>Fill in the <em>Samples</em> sheet <a class="badge bg-secondary" data-bs-toggle="collapse" data-bs-target="#fill-samples">more info</a>
+                <div id="fill-samples" class="panel-collapse collapse">
+                <ol>
+                    <li><strong>IMPORTANT</strong> - Ensure that the <em>Sample Name</em> column is consistent with the
+                        sample names in your AccuCor files.
+                </li>
+                <li>Fill in additional sample details, making sure that the <em>Tissue</em> values
+                    match one of those in the <em>Tissue</em> sheet and the <em>Animal ID</em> values match
+                    one of those in the <em>Animals</em> sheet.
+                </li>
+                </ol>
+                </div>
+            </li>
+            <li>Fill in <em>Treatments</em> sheet (optional) <a class="badge bg-secondary" data-bs-toggle="collapse" data-bs-target="#fill-treatments">more info</a>
+                <div id="fill-treatments" class="panel-collapse collapse">
+                <ol>
+                    <li>Provide a list of treatments used in the study, along with a description
+                        of each.</li>
+                </ol>
+                </div>
+        </li>
+    </ol>
+    <h5 id="optional-validate-your-data">Optional: Validate your data</h5>
+    <p>In order to expedite the loading process, you can check your data submission to
+        resolve simple discrepancies by running your files through our <a href="{% url 'validate' %}">automated
+            validation script</a>.</p>
     <ol>
-        <li>
-            <h5>Prepare your data</h5>
-            <ul>
-                <li>Create a (Google Sheet or Excel) copy of the <a href="https://docs.google.com/spreadsheets/d/1To3495KxJkAtnAD9KVdzc162zKbNiYuPEVyPseQPjMQ/edit?usp=sharing" target="_blank">Tracebase Animal and Sample Table Template</a></li>
-                <li>Ensure that the sample names of the above file are consistent with your Accucor file</li>
-            </ul>
+        <li>Download your Animal and Sample Table in Excel format.</li>
+        <li>Submit the table and any AccuCor files on the <a href="{% url 'validate' %}">TraceBase validation page</a></li>
+        <li>Fix any errors reported, re-download the table and validate the new Excel file.<br/>
+            Note that in general, you should <strong>NOT</strong> edit your AccuCor
+            files. It&#39;s simplest and often best to resolve issues by
+            editing your animal and sample table file.
```

Comment:
NB. Streamline verbiage, clarify instruction (i.e. explicitly where to make the changes), and deemphasize notes to not divert attention from the important bits.
```suggestion
        <li><a href="{% url 'validate' %}">Validate</a> your sample table and AccuCor files</li>
        <li>If there are any errors reported, fix them in the google spreadsheet and return to step 1.<br/>
            <div class="text-muted">Note: In general, you should <strong>NOT</strong> edit your AccuCor
            files. It&#39;s simplest and often best to resolve issues by
            editing your animal and sample table file.</div>
```